package.json

```diff
@@ -78,6 +78,9 @@
     "strip-ansi": "6.0.1",
     "@storybook/react": {
       "trim": "0.0.3"
+    },
+    "default-browser-id": {
+      "meow": "6.1.0"
```

Comment:
is this needed for the `trim-newlines` update?

Reply:
yeah... the `trim-newlines` in question is actually one of the dependencies of `meow`